exporter/logzioexporter/config.go

```diff
@@ -0,0 +1,35 @@
+// Copyright -c Google LLC
```

Comment:
Please use OpenTelemetry copyright notice (I assume you are the author of the code). See recommendations here https://github.com/open-telemetry/community/blob/master/CONTRIBUTING.md#copyright-notices